lib/spree_related_products.rb

```diff
@@ -4,3 +4,4 @@
 require 'spree_related_products/version'
 require 'sass/rails'
 require 'spree_extension'
+require 'deface'
```

Comment:
We should drop deface support I think.

Reply:
If I drop deface support than we can't override views from spree with it and I would need to do it manually by duplicating all the code from the view.